### PR TITLE
feat(delayed_api): support hocon schema

### PR DESCRIPTION
**Pull request by zhongwencool**
1.delayed http api support hocon schema.
2. swagger parameters support ref.

https://github.com/inaka/cowboy_swagger/pull/140

### CODE REVIEW

File: apps/emqx_dashboard/src/emqx_swagger.erl [hunk 1]
```diff
@@ -0,0 +1,43 @@
+-module(emqx_swagger).
```
Comment (zmstone) [inline apps/emqx_dashboard/src/emqx_swagger.erl:1]: this module name is probably not well enough namespaced

Comment (zhongwencool) [reply apps/emqx_dashboard/src/emqx_swagger.erl:1]: rename to emqx_dashboard_swagger_util  or emqx_dashboard_swagger_ref?

**Comment by zhongwencool** [comment]
@stone @DDDHuang should we use cluster_call to replace the rpc_call?

**Comment by zhongwencool** [comment]
we need remove `/mqtt/delayed`. it's a config update.
rename to mgnt_api_config:
/configs/delayed

File: apps/emqx_dashboard/src/emqx_swagger.erl (hunk 1 above)
Comment (zmstone) [reply apps/emqx_dashboard/src/emqx_swagger.erl:16]: `emqx_dashbaord_swagger` should be enough.

**Comment by zhongwencool** [comment]
Ping @zmstone 
the PR is ready, 
I will put `/mqtt/delayed` to /configs/xxx on the next PR.

**Review by zmstone — DISMISSED** [review]

**Review by DDDHuang — APPROVED** [review]

**Pull request merged**